WordPress/Classes/Stores/RemoteFeatureFlagStore.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import WordPressKit
+
+class RemoteFeatureFlagStore {
+
+    init() {
+        DDLogInfo("🚩 Remote Feature Flag Device ID: \(deviceID)")
+    }
+
+    /**
+     Fetches remote feature flags from the server.
+
+     - Parameter callback: An optional callback that can be used to update UI following the fetch. It is not called on the UI thread.
+    */
+    public func update(then callback: FetchCallback? = nil) {
+        // We currently use an anonymous connection to WordPress.com, because we don't need user state.
+        // In the future, we could provide login credentials to the endpoint, which would allow customizing flags server-side on a per-user basis.
+        let remote = FeatureFlagRemote(wordPressComRestApi: WordPressComRestApi.defaultApi())
+        remote.getRemoteFeatureFlags(forDeviceId: deviceID) { [weak self] result in
+            switch result {
+                case .success(let flags):
+                    self?.cache = flags.dictionaryValue
+                    DDLogInfo("🚩 Successfully updated local feature flags: \(flags)")
+                    callback?()
+                case .failure(let error):
+                    DDLogError("🚩 Unable to update Feature Flag Store: \(error.localizedDescription)")
+            }
+        }
+    }
+
+    /// Checks if the local cache has a value for a given `FeatureFlag`
+    public func hasRemoteValueForFlag(_ flag: FeatureFlag) -> Bool {
```

Comment:
I'd consider going for a more Swifty:

```swift
public func hasRemoteValue(for flag: FeatureFlag) {
```

Which would result in calls like this one:

```swift
remoteFeatureFlagStore.hasRemoteValue(for: .newLoginFlow)
```

Reply:
Addressed in 720d35f707545fff468ef82df7c0990ce22d8efa. I also changed the method name slightly – I realized I wasn't happy with one being `hasRemoteValue` and the other being `getValue`, so now it's `hasValue` and `getValue` for consistency.